src/components/AllReviews.jsx

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from "react";
+import { getReviews } from "../api";
+
+export const AllReviews = () => {
+    const [reviews, setReviews] = useState([]);
+
+    useEffect(() => {
+        getReviews().then((review) => {
+            setReviews(review);
+        })
+    }, [])
+
+
+    return <div className="flex">
```

Comment:
Potentially we could use better semantic html to explain what's happening here rather than a div